### DISCUSSION TITLE
[FEAT] XP Boost (Temporary feature)

### DISCUSSION THREAD

**Discussion by Life-Is-Great** [sunflower-land/sunflower-land#1352] [Ideas]
```
Problem:
Expansion is coming soon.
Many players will not be able to gain xp for skill badges because of lacking time and lacking ways to gain xp.
This can lead to many players that are close to achieve those skill badges being unhappy.

```
### Solution 1:
**Trader comes to the village and gives bags of xp for resources (3x xp of what farming those resources would give):**
* Farming
   - **[Big XP Bag]** Players pay 30 wheat,  50 parsnip, 100 carrot & 400 sunflowers (552 XP)
   - **[Small XP Bag]**  Players pay 10 wheat, 20 carrots, 100 sunflowers (102 XP)
* Tools
   - **[Big XP Bag]**  Players pay 1 gold, 5 iron, 10 stone & 100 wood (120 XP)
   - **[Small XP Bag]** 2 iron, 5 stone & 50 wood (49.5 XP)

`If this option is chosen. When purchasing bag of xp -> XP is automatically added to players farm`

### Solution 2:
**Xp cake (Same as "Solution 1" but you bake a cake instead of trader giving you bags of xp):**
* Farming cake
![image](https://user-images.githubusercontent.com/108883042/185815334-12f05f3c-6bbd-472c-af5d-5a87a2dd027b.png)
![image](https://user-images.githubusercontent.com/108883042/185815331-69bbe8d1-26ee-4afd-9f7b-ea5e46e681cb.png)
![image](https://user-images.githubusercontent.com/108883042/185815324-57e317c7-6bed-427f-97dc-0366c9bb3b06.png)

* Tools cake
![image](https://user-images.githubusercontent.com/108883042/185815353-72d9aea2-bbb4-44dc-b5fe-32669ae396a2.png)
![image](https://user-images.githubusercontent.com/108883042/185815359-21ee2de7-2303-4c81-9d2f-bf8a486ebafd.png)
![image](https://user-images.githubusercontent.com/108883042/185815368-48c32d2b-81b8-4eb4-81c9-83a525d47a42.png)

`If this option is chosen. When selling cake -> SFL from farm can go negative instead of positive (That way SFL is also burned)`

### What does SFL win on this?
- [x] **Burned resources**
- [x] **Less fud & complains**
- [x] **More happy players**
- [x] **More balanced SFL economy** (If more players gain xp badges prices will stabilise.)

### What do PLAYERS win on this?
- [x] **Higher selling prices** (Leads to adding more SFL to circulation)
- [x] **Chance not to miss out on never coming back feature** (Locking more long term players)
- [x] **More balanced in game economy** (Leads to less price manipulation by players)
- [x] **Chance to being part of making SFL economy stronger**

### Solution 3:
**When someone clicks on "Wicker man" they get to chose how much and what resources that they want to burn in exchange for xp:**
Same as above. They get 3x xp of what those resources would give to mine/chop/harvest.

<br><br><br><br>

Discord:
GodIsGreat#1718

Farm nr:
146323

**Comment by Tokenergy** [comment]
I think it's a brilliant idea for us who are still far away from decent XP and for burning sfl.

**Comment by sfwhat** [comment]
I don't think a last minute boost to make everyone happy is a great solution. 
If your xp is not enough it is what it is. I am also far away from lvl 10 and after land expansion and public live version we will have enough time to get those achievements.

**Comment by hujikol** [comment]
we need to make this happen, with that we should readjust and recalculate what is needed to trade or craft materials on cakes that equal to a certain amount of XP while it also equals to current XP produced by manually gathering the resource.

**Reply by hujikol** [reply]
Consider this data to further calculate what is needed.

### Farming Level : 
experience 5000+ -> Level 10
experience  3000+ -> Level 9
experience  2000+ -> Level 8
experience  1400+ -> Level 7
experience  1100+ -> Level 6
experience 700+ -> Level 5
experience 350+ -> Level 4
experience 150+ -> Level 3
experience 50+ -> Level 2

### Experience Farming : 
Sunflower: 0.01,
Potato: 0.03,
Pumpkin: 0.1,
Carrot: 0.15,
Cabbage: 0.25,
Beetroot: 0.5,
Cauliflower: 1,
Parsnip: 1.5,
Radish: 2,

### Experience Tool 
Gold Mining: 5 EXP per rock
Iron Mining: 2 EXP per rock
Stone Mining: 0.5 EXP per rock
Wood chopping:  0.2 EXP per tree

**Reply by Life-Is-Great** [reply]
Thank you m8 for your data and support <3

**Comment by ShiningMinions** [comment]
For what it's worth, I would happily burn my resources in exchange for some XP.

**Comment by Renz198** [comment]
Why change the rule on how exp is acquired? Exp is gained by taking action in the game not burning resources. Fake exp like this doesnt make sense. It should stay in grind zone not pay to boost exp.

Why the need to grind exp anyway if someone in the future will just complain and pay to get exp.

**Reply by Life-Is-Great** [reply]
Simple answer to that would be:
Because how is acquired is already set in stone to be changed.

Calling it fake xp is like calling a "Kuebikos" plants fake plants... Or "foreman beavers" wood fake wood... Or "gold rush" gold fake gold etc etc.
It is a **feature** that helps game economy and creates more **long term players** that are caring enough for this game today to burn resources.
This is not changing how xp work. It will only give players opportunity to help SFL economy and balance out the market in exchange for some resources. Just like when you go to minting nft that gives you something in return. (XP boost or whatever)

**Reply by Renz198** [reply]
Paying to get boost was only for gaining resources so it was not fake. You are basically suggesting you can pay for a diploma skipping the hard work as long as you have money. The key word is "hard work" or experience by doing something in the long run not paying to skip it.

**Reply by ShiningMinions** [reply]
I think there may perhaps be some unspoken assumptions being made, in which should be discussed.

> Paying to get boost was only for gaining resources so it was not fake. You are basically suggesting you can pay for a diploma skipping the hard work as long as you have money. The key word is "hard work" or experience by doing something in the long run not paying to skip it.

I think the assumption being made here is that this proposed feature will be permanent, and have no XP cap. Assuming that to be true, then in that case, I agree 100% with you. But what if this was a temporary event? And/or, what if you could only buy a certain amount of XP, and then it stopped? I think there is a really good balance point and doesn't have to necessarily be one extreme or the other.

For a good case study, look at Old School Runescape. You can get XP in so many other ways besides just training in the skill - and it works well and makes the game 10000X more fun. For example: quests. You can get tons of free XP just for completing quests, but in order to complete quests, you need to usually burn some resources. And once you complete the quest, you cannot get the XP reward again because you already completed the quest.

**Reply by Renz198** [reply]
Whether there is cap, time limited doesnt change the fact that it is pay to skip. I could just slack in school as long as i have money I could graduate, is that fair for everyone who did the work (without even a reward) and make exception for few people because they have money (now that have reward)?

**Reply by Life-Is-Great** [reply]
Hello m8.
I understand that getting to this skill was not an easy task.

### Just try to se this from another perspective:
Today. You can pay for:
* Seeds to grow and sell corps +gain xp
* Tools to mine resources +gain xp
* Chickens to get eggs
* Chicken coop to get more eggs than others not having $ or not believing in this game enough to invest
* NFT's to gain free, faster & more xp and Kuebiko to get free seeds
* Resources to craft NFT's
* Potions to grow crops faster
* Buy a farm with high lvl of xp
* Provide liquidity to get free SFL back (Making a wish)

As you se. This game type is called "Pay to own" and giving a chance to players so that they can "Pay to own" and not feel left behind isn't that bad. Specially when you take a look on the gains SFL will make by burning resources.
Plz try to have some common sense and try to se the bigger picture.


If this above isn't enough then try to explain for my son why you don't think that he deserves a chance to gain badges and make him believe that you are fair.

**Reply by ShiningMinions** [reply]
> Whether there is cap, time limited doesnt change the fact that it is pay to skip. I could just slack in school as long as i have money I could graduate, is that fair for everyone who did the work (without even a reward) and make exception for few people because they have money (now that have reward)?

Yeah I definitely see where you're coming from. I don't agree with you, but I 100% understand and respect what your saying. TBH, this is a political discussion - and arguments like these have existed since the beginning of time. I suspect we will not come into agreement, but maybe some common-ground can at least be found.

My first rebuttal is that your argument is an emotional one. "It's not fair" is not a good argument to me. Let's discuss a situation where everybody can win, and the individual + the economy can be benefited as a whole. In my opinion, this proposal is a win-win situation **IF implement properly** (i.e. XP capped + time limited).

My second rebuttal would be that comparing school graduations VS the SFL video-game is a poor comparison. In the context of school, I 100% agree with you. You shouldn't be allowed to "pay to graduate". The reason for this is that your degree should represent some level of understanding you have obtained by attending school. If you present your degree to an employer, the employer hires you, and then employer realizes you have no knowledge to back-up your degree, then that is a very bad thing. That employer got scammed by hiring an incompetent employee.

On the other hand, buying XP in SFL is a completely different situation. I'm not learning anything for a future employer by grinding gold/wood/stone every day. And in fact, I would be doing the game a favor by burning some of the inflated resources I have been hoarding. Please consider this alternative analogy:

A "poor student" and a "rich student" attend school. The "rich student" is able to buy a laptop, and can get homework done faster than the "poor student". This is because the "poor student" cannot afford a laptop, and therefore needs to take a special trips to the library every day to use the computer lab. It takes the "poor student" more time to complete homework.

In this scenario, the rich student paid to get a leg-ahead. Does that mean we should not allow students to use laptops, because it's "not fair" for the "poor student"? If you said "yes", then this is a dangerous and envious way to think. Instead, we should be thinking "how can we also get a laptop for the poor student, in a way that doesn't take from the rich student, and is also good for the school?"

**Reply by esp123** [reply]
I also support the idea that instead of just crafting cakes/foods or crafting xp bags, why not propose a limited time event for new players(exclusively for new players) which give them extra gameplay for x amount of time. In that way, the xps wont look like they just bought it, but they work for it (kinda)...

**Reply by Life-Is-Great** [reply]
That is not bad idea.
But for that to happen more work is required from development team and they do not have much time right now due to work on expansion.

Also, when you think about it. Players do work to gain resources. They have to use their resources for something.

**Comment by esp123** [comment]
Xp bags should be limited time and also limited till you reach 5k xp/lvl10, after that you can't buy anymore xp bags. Or also make different type of cakes/foods which requires diff resources that give diff exps. Also consider x2 xp boost than straight xp gain...

**Comment by davidjr562** [comment]
Has anyone thought about adding this as a feature to the Wicker man?  Burn resources and SFL in exchange for XP?

**Reply by Life-Is-Great** [reply]
I was actually thinking at suggesting this as 3d option.

**Reply by Life-Is-Great** [reply]
Just added this. Thank you for your comment m8 <3
